src/deadline/client/ui/dialogs/submit_job_progress_dialog.py

```diff
@@ -361,3 +349,113 @@ def _build_ui(self):
 
         self.layout.addWidget(self.progress_bar)
         self.layout.addWidget(self.progress_message)
+
+
+class ScrollableWarningDialog(QDialog):
```

Comment:
Two things:

1.   nit: maybe we can name this class `JobSumissionWarningDialog`

    While the message is supplied by the caller, it has the ability to change the auto-accept setting which affects future job submissions.
2.  As it stands currently, this will become part of the public API. Is that intended? Do we want consumers of the python library to use this?

Reply:
Renamed, added a prefixed `_` so that this is not part of the public API, since the public API should go through  the `def handle_request_warning_dialog(self, message: str, default_response: bool):` callback